core/src/Cuda/Kokkos_Cuda_Instance.cpp

```diff
@@ -135,7 +134,6 @@ Kokkos::View<uint32_t *, Kokkos::CudaSpace> cuda_global_unique_token_locks(
   return locks;
 }
 
-// FIXME_CUDA_MULTIPLE_DEVICES
 void cuda_device_synchronize(const std::string &name) {
   Kokkos::Tools::Experimental::Impl::profile_fence_event<Kokkos::Cuda>(
```

Comment:
`cuda_device_synchronize` is only used in the `impl_static_fence` and thus is now interpreted as "synchronize all devices".